install-dev/data/xml/configuration.xml

```diff
@@ -865,5 +865,14 @@ Country</value>
     <configuration id="PS_LOGS_EMAIL_RECEIVERS" name="PS_LOGS_EMAIL_RECEIVERS">
       <value>pub@prestashop.com</value>
     </configuration>
+    <configuration id="PS_SHOW_LABEL_OOS_LISTING_PAGES" name="PS_SHOW_LABEL_OOS_LISTING_PAGES">
+      <value>1</value>
+    </configuration>
+    <configuration id="ADDONS_API_MODULE_CHANNEL" name="ADDONS_API_MODULE_CHANNEL">
+      <value>stable</value>
+    </configuration>
```

Comment:
That should not be in your PR 🤔

Reply:
No, I added it to see if it solved the conflict since it indicated that my line would replace those two entries.

I removed them and left them as they were.

Reply:
I already solved it, sorry